app/src/main/java/com/napzak/market/presentation/explore/ExploreRoute.kt

```diff
@@ -0,0 +1,232 @@
+        onSearchBoxClick = onSearchBoxClick,
+        onTradeTypeClick = { viewModel.changeTradeType(it) },
+        onGenreListClick = { /* TODO: 장르 검색 bottomSheet */ },
+        onSoldOutClick = { viewModel.changeSoldOut() },
+        onUnopenClick = { viewModel.changeUnopen() },
+        onSortButtonClick = { /* TODO: 정렬 bottomSheet */ },
+        onItemClick = { navigateToProductDetail() },
+        onLikeClick = { viewModel.changeItemLikeButton(it) },
+    )
+}
+
+@Composable
+fun ExploreScreen(
+    modifier: Modifier = Modifier,
+    uiState: ExploreUiState,
+    onSearchBoxClick: () -> Unit,
+    onTradeTypeClick: (String) -> Unit,
+    onGenreListClick: () -> Unit,
+    onSoldOutClick: () -> Unit,
+    onUnopenClick: () -> Unit,
+    onSortButtonClick: () -> Unit,
+    onItemClick: (Int) -> Unit,
+    onLikeClick: (Int) -> Unit,
+) {
+    when (uiState.loadState) {
+        is UiState.Loading -> {
+        }
+
+        is UiState.Empty -> {
+        }
+
+        is UiState.Failure -> {
+        }
+
+        is UiState.Success -> {
+            with(uiState.loadState.data) {
+                ExploreSuccessScreen(
+                    modifier = modifier,
+                    tradeType = uiState.tradeType,
+                    genreList = uiState.genreList,
+                    isOnSale = uiState.isOnSale,
+                    isUnopened = uiState.isUnopened,
+                    productList = uiState.loadState.data.productList,
+                    sortType = uiState.sortType,
+                    onSearchBoxClick = onSearchBoxClick,
+                    onTradeTypeClick = onTradeTypeClick,
+                    onGenreListClick = onGenreListClick,
+                    onSoldOutClick = onSoldOutClick,
+                    onUnopenClick = onUnopenClick,
+                    onSortButtonClick = onSortButtonClick,
+                    onItemClick = onItemClick,
+                    onLikeClick = onLikeClick,
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun ExploreSuccessScreen(
+    modifier: Modifier = Modifier,
+    tradeType: String,
+    genreList: List<String>,
+    isOnSale: Boolean,
+    isUnopened: Boolean,
+    productList: List<ProductItem>,
+    sortType: String,
+    onSearchBoxClick: () -> Unit,
+    onTradeTypeClick: (String) -> Unit,
+    onGenreListClick: () -> Unit,
+    onSoldOutClick: () -> Unit,
+    onUnopenClick: () -> Unit,
+    onSortButtonClick: () -> Unit,
+    onItemClick: (Int) -> Unit,
+    onLikeClick: (Int) -> Unit,
+) {
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .background(color = NapzakMarketTheme.colors.white)
+            .padding(top = 40.dp)
+    ) {
+        SearchBox(
+            placeholder = stringResource(id = R.string.explore_search_box_placeholder),
+            readOnly = true,
+            modifier = Modifier
+                .padding(horizontal = 20.dp)
+                .noRippleClickable { onSearchBoxClick() },
```

Comment:
여기서도 람다를 사용하지 않아도 됩니다.
```suggestion
                .noRippleClickable (onSearchBoxClick),
```